recipes/autouri/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "autouri" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00caea88281a2e527e769619cab9c9ecf66baa553cc95203247e8d35526c708c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
```

Comment:
The `setup.py` says that python >=3.6 is required, although I'm not sure if this is strictly true:

https://github.com/ENCODE-DCC/autouri/blob/master/setup.py#L10

Reply:
When I built `autouri` I thought that it requires python >=3.6 due to type hints, but I found that it actually requires python >=3.5. Do I need to rebuild it just to fix it in `setup.py`?

Reply:
Also, I got in docker build of `conda-forge`'s CI, it failed due to a missing package `filelock`, which is already included in `requirements.run` in `meta.yaml`.

```
        from filelock import BaseFileLock
    ModuleNotFoundError: No module named 'filelock'
````

I found this in the same log. `filelock` is already installed.
```
# packages in environment at /opt/conda:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       0_gnu    conda-forge
anaconda-client           1.7.2                      py_0    conda-forge
attrs                     19.3.0                     py_0    conda-forge
beautifulsoup4            4.8.2            py37hc8dfbb8_1    conda-forge
bzip2                     1.0.8                h516909a_2    conda-forge
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2019.11.28       py37hc8dfbb8_1    conda-forge
cffi                      1.14.0           py37hd463f26_0    conda-forge
chardet                   3.0.4           py37hc8dfbb8_1006    conda-forge
click                     7.1.1              pyh8c360ce_0    conda-forge
clyent                    1.2.2                      py_1    conda-forge
conda                     4.8.3            py37hc8dfbb8_1    conda-forge
conda-build               3.19.1           py37hc8dfbb8_0    conda-forge
conda-env                 2.6.0                         1    conda-forge
conda-forge-ci-setup      2.6.0            py37hc8dfbb8_0    conda-forge
conda-forge-pinning       2020.03.30                    0    conda-forge
conda-package-handling    1.6.0            py37h8f50634_2    conda-forge
cryptography              2.8              py37hb09aad4_2    conda-forge
curl                      7.68.0               hf8cf82a_0    conda-forge
decorator                 4.4.2                      py_0    conda-forge
expat                     2.2.9                he1b5a44_2    conda-forge
filelock                  3.0.10                     py_0    conda-forge
```

Reply:
Autouri indeed requires python >=3.6 due to variable annotation. So I changed required version on the recipe from >=3.5 to >=3.6. But got the same `filelock` missing package error.

I do have `filelock` define in `requirements.run`. In order to pass the test, do I also need to define it somewhere else in the `meta.yaml`?
```
requirements:
  host:
    - python
    - pip
  run:
    - python >=3.6
    - requests
    - pyopenssl
    - google-cloud-storage
    - boto3
    - awscli
    - dateparser
    - filelock
```